meta-ibm/recipes-phosphor/chassis/mux-workarounds/mux-workarounds.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Perform workarounds for I2C Muxes
+Wants=sysinit.target
+After=sysinit.target
+
+[Service]
+ExecStart=/usr/bin/env mux-workarounds.sh
+SyslogIdentifier=mux-workarounds
+Type=oneshot
+RemainAfterExit=yes
+
+[Install]
+RequiredBy=basic.target
```

Comment:
basic.target is not something we use often, I guess you're doing this because you want this to run before any of hte other normal user space apps from the multi-user.target?

Reply:
That's correct, especially before any users start grabbing GPIOs

Reply:
Can we not just express the actually dependencies rather than hand-waving with `basic.target`?

Reply:
There are no dependencies though